site/package.json

```diff
@@ -11,7 +11,7 @@
   "dependencies": {
     "@xstate/inspect": "^0.7.0",
     "@xstate/react": "^3.0.1",
-    "@userfront/toolkit": "^1.0.0-alpha.17",
```

Comment:
What was the motivation for using the remote package instead of the local one?

Reply:
It is the local one, actually! [`npm workspaces` automatically symlinks compatible libraries within a workspace.](https://docs.npmjs.com/cli/v10/using-npm/workspaces)